docs/reference/api/widgets/table.rst

```diff
@@ -64,25 +64,14 @@ to control the display order of columns independent of the storage of that data.
     )
 
     # Get the details of the first item in the data:
-    print(f"{table.data[0].name}, who is age {table.data[0].age}, is from {table.data[0].planet}")
+    print(f"{table.data[0].name}, who is age {table.data[0].age}, "
+          f"is from {table.data[0].planet}")
 
-The attribute names used on each row of data (called "accessors") are created automatically from
-the name of the headings that you provide. This is done by:
```

Comment:
I moved some details into the constructor docstrings to make them self-contained, leaving the Usage section as mostly just examples.